tests/test_util.py

```diff
@@ -408,6 +408,16 @@ def test_get_manifest_digests_missing(tmpdir, has_content_type_header, has_conte
 
     url = 'https://example.com/v2/spam/manifests/latest'
 
+    def custom_retries(*args, **kwargs):
```

Comment:
Does this functions makes retries skip the function at all? Why is it needed here?